src/arrayMethodSort.js

```diff
@@ -4,9 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    if (`${a}` < `${b}`) {
+      return -1;
+    } else if (`${a}` > `${b}`) {
+      return 1;
+    } else {
+      return 0;
+    }
+  }) {
```

Comment:
1. Move it into separated function
2. use early return pattern
```suggestion
   const defaultCompareFunction = (a, b) => {
    if (`${a}` < `${b}`) {
      return -1;
    } 
    
    if (`${a}` > `${b}`) {
      return 1;
    } 

    return 0;
  }
  
  [].__proto__.sort2 = function(compareFunction = defaultCompareFunction) {
```